war/pom.xml

```diff
@@ -106,33 +106,59 @@ THE SOFTWARE.
       <version>5.13</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.modules</groupId>
-      <artifactId>instance-identity</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>ssh-cli-auth</artifactId>
```

Comment:
Note that this (and `sshd`) pick up a transitive dep on `instance-identity 2.2` which then still gets packaged into the WAR as a module. Cannot fix until #5049 is merged (or this is merged into that).